backend/routes/service.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const { models } = require("../sequelize");
+const extractFiles = require("../middlewares/files");
+const files = require("../middlewares/files");
+
+const router = express.Router();
+
+// {TODO(Arjan): add exception handling}
+router.post("", extractFiles, async (req, res) => {
+  const { body, files } = req;
+
+  const newService = await models.service.create({
+    category: body.category,
+    name: body.name,
+    estimateUnit: body.estimateUnit,
+    priceEstimate: body.priceEstimate,
+    description: body.description,
+    contact: body.contactNumber,
+  });
+
+  locations = body.locations.split(",");
+  // add all the locations
+  // associate service with all the locations
+  for (let i = 0; i < locations.length; ++i) {
+    let loc = await models.location.findByPk(locations[i]);

```

Comment:
Will this still work? As location name is not private key anymore.

Reply:
Right, it won't. Working on fixing it

Reply:
Fixed in latest commit